assets/js/vendor/index.js

```diff
@@ -3,3 +3,12 @@
 require('nodep-date-input-polyfill');
 
 require('jquery.mmenu');
+
+require('tooltipster');
+
+$(document).ready(function() {
+  $('.hastip').tooltipster({
+     trigger: 'click',
```

Comment:
We can have hover for mouse and tap for touch, if that's what you're alluding to. Having hover and click for mouse seems unnecessary.